app/src/main/java/com/example/sharingang/utils/notification/MyFirebaseMessagingService.kt

```diff
@@ -21,9 +23,22 @@ class MyFirebaseMessagingService: FirebaseMessagingService() {
         remoteMessage.data.let {
             val userId = remoteMessage.data["userId"]
             val deeplink = remoteMessage.data["deeplink"]
-            if (userId != currentUserProvider.getCurrentUserId() && deeplink != null) {
+            var channelId = ""
+            val title = when (remoteMessage.data["notificationType"]) {
+                "new_item" -> {
+                    channelId = NotificationFields.NEW_ITEM_CHANNEL_ID
+                    applicationContext.getString(R.string.new_item_notification_message)
+                }
+                "chat" -> {
+                    channelId = NotificationFields.CHAT_CHANNEL_ID
+                    remoteMessage.data["fromName"]+":"
+                }
```

Comment:
Is it not possible to use the constants you defined earlier for comparison here?

Reply:
The problem is that those are used first in functions/index.js file, and I haven't found a way to use constants from another file type... If you know how I'll gladly change it

Reply:
Ah, right, makes sense